docs/oscar_references.bib

```diff
@@ -98,6 +98,36 @@ @MastersThesis{Bhm99
   year          = {1999}
 }
 
+@article{Blumenhagen:2010pv,
+   title={Cohomology of line bundles: A computational algorithm},
+   volume={51},
+   ISSN={1089-7658},
+   url={http://dx.doi.org/10.1063/1.3501132},
+   DOI={10.1063/1.3501132},
+   number={10},
+   journal={Journal of Mathematical Physics},
+   publisher={AIP Publishing},
+   author={Blumenhagen, Ralph and Jurke, Benjamin and Rahn, Thorsten and Roschy, Helmut},
+   year={2010},
+   month={Oct},
+   pages={103525}
+}
+
+@article{Blumenhagen:2010ed,
+   title={Cohomology of line bundles: Applications},
+   volume={53},
+   ISSN={1089-7658},
+   url={http://dx.doi.org/10.1063/1.3677646},
```

Comment:
```suggestion
   url={https://doi.org/10.1063/1.3677646},
```
And so on...

Reply:
Please also follow https://oscar-system.github.io/Oscar.jl/dev/DeveloperDocumentation/build_documentation/#Updating-the-bibliography

Reply:
Thx. Adjusted.

For the @ misc  entry concerning the implementation of `cohomCalg`, `bibtool` produces no key. I use the one suggested on https://github.com/BenjaminJurke/cohomCalg, i.e. `cohomCalg:Implementation`.